.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,97 @@
+name: Build and Test
```

Comment:
@armfazh can you take a look at this workflow? Anything you'd port over from CIRCL or does it look good as is?

Reply:
script is ok.